### DISCUSSION TITLE
Can I configure expiry validation?

### DISCUSSION THREAD

**Discussion by jordaniversen** [golang-jwt/jwt#186] [General]
I was previously browsing the code and thought there was a way to do this, but I can't seem to figure it out.  Is there a way to parse a JWT token and configure the validation, such as not checking "exp" or other tags?

Thanks!

**Comment by jordaniversen** [comment]
I can't seem to find a way to delete this discussion, so I'll add an update.  The answer is to use `jwt.WithoutClaimsValidation()`.

**Reply by mfridman** [reply]
Please be careful, as this sets `SkipClaimsValidation = true` and disables validation. This might be what you're after, but a disclaimer nonetheless.
